pkgs/development/python-modules/flask-mongoengine/default.nix

```
@@ -15,18 +15,20 @@
 
 buildPythonPackage rec {
   pname = "flask-mongoengine";
-  version = "1.0.0";
+  version = "unstable-2023-11-10";
```

```suggestion
  version = "1.0.0.-unstable-2023-11-10";
```

There are new guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

Ah. Good catch. What about the SETUPTOOLS_SCM_PRETEND_VERSION? When playing around with it, it seemed to strip anything but the major, minor and patch triple. Is 1.0.9999 okay, or should it be 1.0.0? Or 1.0.1?

---

Or should I just use ${version} and let it do it's thing?

---

Turns out setuptools-scm is not very happy with ${version}. "1.0.0" works.

---

Is this version sent to the upstream via some user tracking? Then it might courteous to indicate this is a patched version.

---

I believe setuptools supports the scheme `MAJOR.MINOR.PATCH+LOCAL`